CONTRIBUTING.md

```diff
@@ -1,15 +1,86 @@
 # Contributing to StoreFront
 
-When contributing to StoreFront, make a branch for your change named
-with the ticket number. As part of your change, add an entry to the
-changelog that describes your change. When your pull request is merged
-into `master`, a release is made based on the changelog entry.
+Thank you for your interest in contributing to StoreFront.
 
-## Modify the CHANGELOG.md
+The following is a set of guidelines for contributing to StoreFront.
+Make sure to read the [README](README.md) first to get set up with the
+project.
 
-Our CHANGELOG.md follows the [Keep a Changelog](https://keepachangelog.com)
-format. Add an "Unreleased" entry to the top of the
-changelog in the following format, removing any inapplicable sections:
+## Making a change
+
+StoreFront follows the [GitHub flow][] for branching and merging. All
+work is done on feature branches, which fork from and merge into
+`master`.
+
+[GitHub flow]: https://help.github.com/articles/github-flow/
+
+Preferably, you should have write access to this repo and will make a
+branch directly on it. If you do not, request access or fork the repo.
+
+1. Discuss any non-trivial changes with the StoreFront team before
+   committing to any work to ensure that there are no surprises for
+   either party.
+2. Branch from the tip of `master`. If your change has a Jira ticket
+   associated with it (most changes should), name the branch after the
+   ticket number (e.g. `SF-1156`); if not, name it something
+   descriptive.
+3. Make your change. Your change should build correctly and be fully tested.
+   - Use `yarn dev` in each package that is being changed. This command
+     starts a process that builds the package whenever one of its files
+     changes. Note that the process must be restarted if a file in a
+     dependency changes (e.g. a change in flux-capacitor will not cause
+     core to rebuild).
+   - Each package has unit tests in the package's `test` directory. Be
+     sure to add unit tests for any new code you write and for any
+     existing code you change. There are currently no integration tests,
+     end-to-end tests, or tests for HTML templates.
+   - While developing, you can use the `yarn tdd` command in the package
+     being changed to run the tests whenever a file is saved.
+4. Add a changelog entry in each package that was changed. See the
+   [Modifying the changelog](#modifying-the-changelog) section below.
+5. Push your branch up and make a [pull request](https://help.github.com/articles/creating-a-pull-request/)
+   against the `master` branch. Give your pull request a title in this
+   format: `SF-####: Descriptive title`, where `SF-####` is the Jira
```

Comment:
Do you need to include `SF-` when explaining how to resolve the placeholder? What about: "...where `####` is the Jira ticket number."?

Reply:
Most of the tickets will come from the SF project, but it could technically come from any project. I'll change the wording to "Jira issue key" like the [Atlassian docs](https://confluence.atlassian.com/adminjiracloud/connect-jira-cloud-to-github-814188429.html) refer to it.

Reply:
Sounds good, thanks. Resolving.